src/ng-select/lib/ng-select.component.spec.ts

```diff
@@ -2348,20 +2348,20 @@ describe('NgSelectComponent', () => {
 			expect(fixture.debugElement.query(By.css('.ng-placeholder'))).toBeTruthy();
 		}));
 
-		it('should not display ng-placeholder if an item is selected', fakeAsync(  () => {
+		it('should not display ng-placeholder if an item is selected', fakeAsync(() => {
```

Comment:
The previous thest became "should not display ng-placeholder if an item is selected and fixedPlaceholder is false" 

the component became 

`<ng-select [(ngModel)]="selectedCity" [fixedPlaceholder]="false" 
														 [items]="cities" bindLabel="name" 
														 placeholder="testPlaceholder">			
                  </ng-select>`
and explectation became 

`expect(fixture.debugElement.query(By.css('.ng-placeholder'))).toBeFalsy();`